cmd/flakechecker/bq_client.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"cloud.google.com/go/bigquery"
+	"context"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/api/iterator"
+	"regexp"
+	"time"
+)
+
+const projectID = "acs-san-stackroxci"
+const queryTimeout = 1 * time.Minute
+
+type biqQueryClient interface {
+	GetRatioForTest(flakeTestConfig *flakeCheckerRecord, testName string) (int, int, error)
+}
+
+type biqQueryClientImpl struct {
+	client *bigquery.Client
+}
+
+func getNewBigQueryClient() (biqQueryClient, error) {
+	ctx := context.Background()
+
+	client, err := bigquery.NewClient(ctx, projectID)
+	if err != nil {
+		return nil, errors.Wrap(err, "creating BigQuery client")
+	}
+
+	return &biqQueryClientImpl{client: client}, nil
+}
+
+func getFilteredTestName(name string) string {
+	// Apply the same filtering used in DB to generate test names not influenced with version Z number.
```

Comment:
I think this need more context. Where is the code for DB filtering. Maybe an example of name and filtered name

Reply:
I'll drop this. This will not be needed anymore with: https://github.com/stackrox/stackrox/pull/13185